### PR TITLE
Added code for count of vowels in a string in JavaScript

### LINKED ISSUE #5011: Count number of vowels in a sentence
## 💥 Proposal

I want to add the code for count number of vowels in a sentence inputted by user in JavaScript.[GSSOC'21]
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#reporting-new-issues)?

Yes

### PR DESCRIPTION
**Pull request by SAY-droid427**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Code for counting number of vowels in a string in JavaScript

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Fixes https://github.com/TesseractCoding/NeoAlgo/issues/5011

### CODE REVIEW

File: JavaScript/Algorithms/vowelCount.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+function vowelCount(str){
+    let count=0
+    for(let character of str){
+        if(vowels.includes(character)){
+            count++;
+        }
+    }
+    return count;
+}
+const string="Javascript is necessary for web development"
```
Comment (HarshCasper) [inline JavaScript/Algorithms/vowelCount.js]: Use dynamic input

**Review by HarshCasper — CHANGES_REQUESTED** [review]

File: JavaScript/Algorithms/vowelCount.js (hunk 1 above)
Comment (SAY-droid427) [reply JavaScript/Algorithms/vowelCount.js]: Done

**Comment by SAY-droid427** [comment]
@HarshCasper Please review this PR

File: JavaScript/Algorithms/vowelCount.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    for(let character of str){
+        if(vowels.includes(character)){
+            count++;
+        }
+    }
+    return count;
+}
+prompt("Enter string:")
+const count=vowelCount(str)
+console.log(count)
```
Comment (HarshCasper) [inline JavaScript/Algorithms/vowelCount.js]: This part is buggy. You are not even assigning `str`

**Review by HarshCasper — CHANGES_REQUESTED** [review]

File: JavaScript/Algorithms/vowelCount.js (hunk 2 above)
Comment (SAY-droid427) [reply JavaScript/Algorithms/vowelCount.js]: I have corrected the code

**Comment by SAY-droid427** [comment]
@HarshCasper Please check once more if there are any mistakes.

**Comment by SAY-droid427** [comment]
@HarshCasper Please review this PR

File: JavaScript/Algorithms/vowelCount.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+const vowels=["a","e","i","o","u","A","E","I","O","U"]
+function vowelCount(str){
+    let count=0
+    for(let character of str){
+        if(vowels.includes(character)){
+            count++;
+        }
+    }
+    return count;
+}
```
Comment (HarshCasper) [inline JavaScript/Algorithms/vowelCount.js]: Put this in a function

**Review by HarshCasper — CHANGES_REQUESTED** [review]

File: JavaScript/Algorithms/vowelCount.js (hunk 3 above)
Comment (SAY-droid427) [reply JavaScript/Algorithms/vowelCount.js]: Done

Comment (SAY-droid427) [reply JavaScript/Algorithms/vowelCount.js]: Please review it and merge the PR

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]
Few changes: 

- Add Multi-Line Comments 
- Add a beginner-friendly description at the top
- Use proper code Format 
- Remove extra lines of code and spaces
- Add sample I/O at the end of the Code
- Add Time and Space Complexity at the end of the Code   
- Add new line at the EOF

Use proper display message how you are reading and writing outputs

**Review by ankitaggarwal23 — COMMENTED** [review]
Why do we need such problems in Algorithm section?

**Review by ankitaggarwal23 — COMMENTED** [review]
Closing this due to inactivity. Read the instructions mentioned on Gssoc discord yesterday.

**Pull request closed without merging**